components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_partition.c

```diff
@@ -1284,12 +1274,17 @@ static void psa_key_management_operation(void)
                         SPM_PANIC("SPM read length mismatch");
                     }
 
+                    /* the size of the client side psa_key_id_t is expected to be 4 bytes */
+                    if (msg.in_size[1] != 4) {
```

Comment:
why not make the code self explanatory and do something like this
#define CLIENT_PSA_KEY_ID_SIZE_IN_BYTES = 4 
and then
if (bytes_read != CLIENT_PSA_KEY_ID_SIZE_IN_BYTES )

Reply:
Done